src/components/ProjectionChart.vue

```diff
@@ -1,9 +1,11 @@
 <script>
 import { Line, mixins } from 'vue-chartjs';
+import _ from 'lodash';
```

Comment:
Lamentablemente, dado el estado actual de JS/Webpack una librería así es muy poco eficiente. Te recomiendo importar cada función de manera independiente, `import cloneDeep from 'lodash/cloneDeep'` 

Échale un vistazo a https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark por ejemplo!

Reply:
arreglado